odata-parser.pegjs

```diff
@@ -641,19 +642,17 @@ DurationNumber =
 Text =
 	// This regex is equivalent to `(!ReservedUriComponent)`
 	text:$[^:/?#\[\]@!$*&()+,;=]*
-	{ return decodeURIComponent(text) }
+	{ return text }
 
 Sign =
 		'+'
-	/	'%2B'
 		{ return '+' }
 	/	'-'
 	/	''
 
 // TODO: This should really be done treating everything the same, but for now this hack should allow FF to work passably.
 Apostrophe =
```

Comment:
This should probably be in-lined now that it only checks a single character and there (probably?) won't be any benefit from memoizing, and definitely no benefit from deduplicating